cristin-project/src/test/java/no/unit/nva/cristin/projects/createSampleProject.feature

```diff
@@ -0,0 +1,86 @@
+Feature: API tests for Cristin Project retrieve and search
+
+  Background:
+    * def domainName = java.lang.System.getenv('DOMAIN_NAME')
+    * def basePath = java.lang.System.getenv('BASE_PATH')
+    * def CRISTIN_BASE =  'https://' + domainName +'/' + basePath
+    * def username = java.lang.System.getenv('TESTUSER_FEIDE_ID')
+    * def password = java.lang.System.getenv('TESTUSER_PASSWORD')
+    * def cognitoClientAppId = java.lang.System.getenv('COGNITO_CLIENT_APP_ID')
+    * def cognitoUserpoolId = java.lang.System.getenv('COGNITO_USER_POOL_ID')
+    * def tokenGenerator = Java.type('no.unit.nva.cognito.CognitoUtil')
+    * def token = tokenGenerator.loginUser(username, password, cognitoUserpoolId, cognitoClientAppId)
+    * def swaggerSampleProject =
+    """
+    {
+      '@context': 'https://bibsysdev.github.io/src/project-context.json',
+      'type': 'Project',
+      'title': 'Example Title',
+      'language': 'http://lexvo.org/id/iso639-3/nno',
+      'alternativeTitles': [
+        {
+          'nb': 'Eksempel på tittel'
+        }
+      ],
+      'startDate': '2012-01-09T00:00:00Z',
```

Comment:
add milliseconds with value other than 000. Something strange about Cristin not allowing 000